demos/calibration/calibration-pubmed-link-prediction.ipynb

```diff
@@ -0,0 +1,1795 @@
+{
```

Comment:
<p>I get Value error here:</p><p>`ValueError: X should be a 1d array`</p><p><br></p><p>and the shapes are:</p><p>((15070, 1), (15070,))</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>

Reply:
This has been fixed in the library.